RegExp.make.js

```diff
@@ -37,7 +37,7 @@ RegExp.make = (function () {
     if (0x20 <= n && n <= 0x7e) {
       return String.fromCharCode(n).replace(UNSAFE_CHARS_CHARSET, '\\$&');
     }
-    var hex = n.toString(16);
+    const hex = n.toString(16);
```

Comment:
> `npm install` doesn't seem to install all the dependencies for this JS project - how do I install closure compiler?

Sorry, I missed this comment for some reason.  I assigned a pull request to you earlier today that gets us running via `npm test` and gets the basics for travis checks.  If that looks good in broad strokes, then I'll just have travis ci curl down closure-compiler-latest and run.